docs/contributor-guide.md

```diff
@@ -33,4 +33,10 @@ provided by the bot. You will only need to do this once across all repos using o
 
 This project has adopted the [Microsoft Open Source Code of Conduct](https://opensource.microsoft.com/codeofconduct/).
 For more information see the [Code of Conduct FAQ](https://opensource.microsoft.com/codeofconduct/faq/) or
-contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additional questions or comments.
\ No newline at end of file
+contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additional questions or comments.
+
+## Getting Started
+
+You�ll need Visual Studio 2019 installed to do work in a GUI, or at least the MSBuild and SDK toolchains to build from the command line.  From Visual Studio, open the ProjectReunion.sln and hit build!
```

Comment:
Also add a bit about "Or, start a Developer Command Prompt or Developer Powershell for VS 2019, and use `msbuild ProjectReunion.sln".

(Consider also wrapping text at ~100ch; looks like we need a .editorconfig in the root.)

Reply:
contributor-guide.md had longer lines so I didn't know it was a problem

I'll take a shot at .editorconfig in next commit